### PR TITLE
Fix ComponentModel tests on Linux

### LINKED ISSUE #15931: Test failures in System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String and System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String_InvalidCulture
These tests are currently disabled with label [ActiveIssue(11611)](https://github.com/dotnet/corefx/issues/11611), although the issue has been closed. If you enable the tests, they would still fail on Linux.

```
System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.ArgumentException)
        Actual:   (No exception was thrown)
        Stack Trace:
           /home/sekho/Desktop/corefx/src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs(61,0): at System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String()
     System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String_InvalidCulture [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.ArgumentException)
        Actual:   (No exception was thrown)
        Stack Trace:
           /home/sekho/Desktop/corefx/src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs(81,0): at System.ComponentModel.Tests.CultureInfoConverterTest.ConvertFrom_String_InvalidCulture()
  Finished:    System.ComponentModel.TypeConverter.Tests
```

### PR DESCRIPTION
**Pull request by tarekgh**
Fixes #15931

### CODE REVIEW

**Comment by tarekgh** [comment]
@safern @AlexGhiondea

**Comment by tarekgh** [comment]
CC @sepidehMS @danmosemsft

File: src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs [hunk 1]
```diff
@@ -44,22 +44,30 @@ public void CanConvertTo()
             Assert.True(converter.CanConvertTo(typeof(InstanceDescriptor)));
         }
 
-        [ActiveIssue(11611, TestPlatforms.AnyUnix)]
         [Fact]
         public void ConvertFrom_String()
         {
             CultureInfo c;
-
+
```
Comment (sepidehMS) [inline src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs]: Nit: extra spaces

**Comment by sepidehMS** [comment]
Thanks @tarekgh!

File: src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj [hunk 2]
```diff
@@ -65,8 +65,6 @@
     <Compile Include="UInt32ConverterTests.cs" />
     <Compile Include="UInt64ConverterTests.cs" />
     <Compile Include="UriTypeConverterTests.cs" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetGroup)' == 'netstandard'">
```
Comment (safern) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: Why are we removing this?

Comment (tarekgh) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: it was not needed. the test has netstandard and netcoreapp configurations and both are working fine with these files. @AlexGhiondea may comment on that.

Comment (safern) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: I was just curious! Thanks for clarifying @tarekgh

**Comment by tarekgh** [comment]
The failed job is already tracked by https://github.com/dotnet/corefx/issues/15383

**Pull request merged**